src/components/FullWindowOverlay.tsx

```diff
@@ -10,13 +17,14 @@ const NativeFullWindowOverlay: React.ComponentType<
 > = FullWindowOverlayNativeComponent as any;
 
 function FullWindowOverlay(props: { children: ReactNode }) {
+  const { width, height } = useWindowDimensions();
   if (Platform.OS !== 'ios') {
     console.warn('Using FullWindowOverlay is only valid on iOS devices.');
     return <View {...props} />;
   }
   return (
     <NativeFullWindowOverlay
-      style={{ position: 'absolute', width: '100%', height: '100%' }}>
+      style={[StyleSheet.absoluteFill, { width, height }]}>
```

Comment:
Do we need `absoluteFill` here, wasn't `position: 'absolute', width, height` enough?

Reply:
We still need `top: 0` to ensure correct origin so I decided to utilise the `absoluteFill`.
![simulator_screenshot_1A032E77-EDC4-40F0-A30E-6B934C7ACF78](https://github.com/user-attachments/assets/19a53ef8-8de4-4f69-8832-fb076270be02)

Reply:
Okay, I see. That's fine with me. Just make sure this works as expected - buttons, pressables on full window overlay work correctly. I do not remember whether the buttons under FWO should be clickable, but if so - let's make sure the behaviour is unchanged. 

Provided that above assertions are upheld, I believe we're good to go.